draft-ietf-quic-multipath.md

```diff
@@ -1416,16 +1416,23 @@ PATH_CIDS_BLOCKED frames are formatted as shown in {{fig-path-cid-blocked-frame-
 PATH_CIDS_BLOCKED Frame {
   Type (i) = TBD-09 (experiments use 0x15228c0e),
   Path Identifier (i),
+  Max Sequence Number (i),
```

Comment:
Depending on whether we agree on the "Next Sequence Number" field, this should be updated as well.
```suggestion
  Next Sequence Number (i),
```